README.md

```diff
@@ -13,15 +13,42 @@ Usage of this gem is pretty straight-forward. Add it to your bundle like this:
 gem 'jekyll-github-metadata'
 ```
 
-Then go ahead and run `bundle install`. Once you've done that, add your repo & the gem to your `_config.yml`:
+Now add it to your `_config.yml`:
 
 ```yaml
-# or use PAGES_REPO_NWO in the env
-repository: me/super-cool-project
 gems: ['jekyll-github-metadata']
 ```
 
-Then run `jekyll` like you normally would and your `site.github.*` fields should fill in like normal.
+Then go ahead and run `bundle install`. Once you've done that jekyll-github-metadata will run when you run Jekyll.
+
+In order for jekyll-github-metadata to know what metadata to fetch it must
+be able to determine the repository to ask GitHub about.
+
+The easiest way to accomplish this is by setting an "origin" remote with a
+github.com URL. If you run `git remote -v` in your repository, you should
+see your repo's URL.
+
+Alternatively, you can set the repo NWO as either the `PAGES_REPO_NWO`
+environment variable:
+
+```shell
+$ PAGES_REPO_NWO=username/repo-name bundle exec jekyll build
+```
+
+... or add your repo name to your `_config.yml`:
```

Comment:
Why three options? Why is the ENV var before the config (which users presumably now)?